app/scripts/lib/url.js

```diff
@@ -89,6 +89,16 @@ function (_) {
       return origin;
     },
 
+    /**
+     * Returns true if given "uri" has HTTP or HTTPS scheme
+     *
+     * @param {String} uri
+     * @returns {boolean}
+     */
+    isHTTP: function (uri) {
```

Comment:
Not sure I have a better naming suggestion, but when I saw the name of this method, I immediately thought "oh, it's checking whether the URL is plain http instead of https".